pennylane/pulse/parametrized_evolution.py

```diff
@@ -73,9 +77,21 @@ class ParametrizedEvolution(Operation):
             from comparing a 4th and 5th order Runge-Kutta step in the Dopri5 algorithm. This error
             is guaranteed to stay below ``tol = atol + rtol * abs(y)`` through adaptive step size
             selection. Defaults to 1.4e-8.
-        mxstep (int, optional): maximum number of steps to take for each timepoint for the ODE solver. Defaults to
-            ``jnp.inf``.
+        mxstep (int, optional): maximum number of steps to take for each timepoint for the
+            ODE solver. Defaults to ``jnp.inf``.
         hmax (float, optional): maximum step size allowed for the ODE solver. Defaults to ``jnp.inf``.
+        return_intermediate (bool): Whether or not the ``matrix`` method returns all intermediate
+            solutions of the time evolution ODE at the times provided in ``t = [t_0, t_1, .. , t_f]``. If ``False``
+            (the default), only the matrix for the full time evolution is returned. If ``True``,
+            all solutions including the initial condition :math:`U(t_0, t_0)=1` are returned; when
+            used in a circuit, this results in ``ParametrizedEvolution`` being a broadcasted operation.
```

Comment:
.. see usage details or "Computing intermediate results" below

or something like that

Reply:
Added this. I noticed, there also should be a brief showcase for the intermediate values of a QNode, so I added that to the docstring in the mentioned usage details section.

Reply:
great 👍